openwisp_monitoring/device/api/views.py

```diff
@@ -209,7 +233,10 @@ def _calculate_increment(self, ifname, stat, value):
         # get previous counters
         data = self._previous_data
         try:
-            previous_counter = data['interfaces_dict'][ifname]['statistics'][stat]
+            if ifname is None:
```

Comment:
Are you sure this is correct? if ifname is None, then `data['interfaces_dict'][ifname]['statistics'][stat]` will fail

Reply:
It should be exact opposite, made some last minute changes yesterday. Now I get why those 3 tests where failing lately :sweat_smile: